Platform/OpenCore/OpenCore.c

```diff
@@ -175,13 +185,43 @@ OcBootstrapRerun (
   if (This->NestedCount == 1) {
     mOpenCoreVaultKey = OcGetVaultKey (This);
 
+
+    EFI_LOADED_IMAGE_PROTOCOL* LoadedImage = NULL;
+    Status = gBS->HandleProtocol (
+      gImageHandle,
+      &gEfiLoadedImageProtocolGuid,
+      (VOID **) &LoadedImage
+      );
+
+    if (EFI_ERROR (Status)) {
```

Comment:
Missing Loaded image protocol should not abort the startup, can always fallback to the hardcoded default path.